README.md

```diff
@@ -1,140 +1,75 @@
-      ...
-      ...
-      return true;
-    }
-    
-    @Override
-    public void onScaleEnd(StandardScaleGestureDetector detector) {
-      // revert thresholds values
-      RotateGestureDetector rotateGestureDetector = androidGesturesManager.getRotateGestureDetector();
-      rotateGestureDetector.setAngleThreshold(Constants.DEFAULT_ROTATE_ANGLE_THRESHOLD);
-    }
+```java
+// In the app build.gradle file
+dependencies {
+	implementation 'com.mapbox.mapboxsdk:mapbox-android-gestures:0.3.0-SNAPSHOT'
+}
 ```
 
-## Detectors
-With this library you will be able to recognize gestures using detectors provided by the Support Library and more.
+To run the [sample code](#sample-code) on a device or emulator, include your [developer access token](https://www.mapbox.com/help/define-access-token/) in `developer-config.xml` found in the project. 
 
-#### StandardGestureDetector
-Wraps [GestureDetectorCompat](https://developer.android.com/reference/android/support/v4/view/GestureDetectorCompat.html) exposed via the Support Library that recognizes gestures like tap, double tap or scroll.
+## Getting Help
 
-#### StandardScaleGestureDetector
-Wraps [ScaleGestureDetector](https://developer.android.com/reference/android/view/ScaleGestureDetector.html) exposed via the Support Library that recognizes scale/pinch gesture.
+- **Need help with your code?**: Look for previous questions on the [#mapbox tag](https://stackoverflow.com/questions/tagged/mapbox+android) — or [ask a new question](https://stackoverflow.com/questions/tagged/mapbox+android).
+- **Have a bug to report?** [Open an issue](https://github.com/mapbox/mapbox-gestures-android/issues). If possible, include the version of Mapbox Core that you're using, a full log, and a project that shows the issue.
+- **Have a feature request?** [Open an issue](https://github.com/mapbox/mapbox-gestures-android/issues/new). Tell us what the feature should do and why you want the feature.
 
-#### MultiFingerTapGestureDetector
-Simple gesture detector that notify listeners whenever a multi finger tap occurred and how many fingers where involved.
+## Sample code
 
-#### RotateGestureDetector
-A detector that finds the angle difference between previous and current line made with two pointers (fingers).
+[This repo's test app](https://github.com/mapbox/mapbox-gestures-android/tree/master/app/src/main/java/com/mapbox/android/gestures/testapp) can also help you get started with the Gestures library.
 
-#### ShoveGestureDetector
-Detects a vertical movement of two pointers if they are placed within a certain horizontal angle.
+## Contributing
 
-#### MoveGestureDetector
-Behaves similarly to `#onScroll()` contained in the `StandardGestureDetector`, however, its a `ProgressiveGesture` that enables better filtering options, as well as thresholds.
+We welcome feedback, translations, and code contributions! Please see [CONTRIBUTING.md](CONTRIBUTING.md) for details.## Version
```

Comment:
Could you bundle the `CONTRIBUTING.md` file with this PR?
Also, a typo in `Version` header.

Reply:
file added and typo fixed